cmd/cephcsi.go

```diff
@@ -120,6 +120,11 @@ func init() {
 		"path of prometheus endpoint where metrics will be available")
 	flag.DurationVar(&conf.PollTime, "polltime", time.Second*pollTime, "time interval in seconds between each poll")
 	flag.DurationVar(&conf.PoolTimeout, "timeout", time.Second*probeTimeout, "probe timeout in seconds")
+	flag.DurationVar(
+		&conf.LogSlowOpsAfter,
+		"logslowopsafter",
+		time.Second*15,
```

Comment:
Do we need to enable it by default? how about disable it by default and if the user requires they can enable it?

Reply:
I'm fine with enabling by default, it can help with troubleshooting issues.